source/extensions/quic_listeners/quiche/platform/quic_socket_address_impl.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include <stdint.h>
+#include <sys/socket.h>
```

Comment:
again interested in the portability story.

Reply:
This will be soon implemented using in_addr and in6_addr as be removed out of platform layer.

Reply:
Add TODO?